### DISCUSSION TITLE
Instalação em Ambiente Windows da Locaweb

### DISCUSSION THREAD

**Discussion by RFS-Solutions** [RamonSilva20/mapos#2567] [General]
Bom dia,

Gostaria de saber se é possível instalar o sistema em um ambiente CentOS (Compartilhado) da Locaweb?

Ou, se for o caso, realziar a instalação localhost e posteriormente transferir completamente para a hospedagem?

Agradeço :)

**Comment by Pr3d4dor** [comment]
Sim, é possível.

**Reply by RFS-Solutions** [reply]
Grato por sua resposta. Mas no caso, não Locaweb, hospedagem compartilhada, não se tem acesso Root, que me impede de instalar o docker e o compose.

No caso o sistem da hospedagem é o CentOS.

Saberia informar se criando tudo localhost em ambiente Windows ou Linux e transferindo pra la, funcionaria?

Grato por sua ajuda.

**Reply by Pr3d4dor** [reply]
Funcionaria sim, não precisa de acesso root.

**Comment by RFS-Solutions** [comment]
Resolvido, pessoal da parceira do Maps fez a instalação. Obrigadop Sysgo
